packages/utilities/gel-foundations/src/scripts/typographyConfig.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-useless-path-segments */
+import arabicConfig from '../../src/scripts/arabic.json';
+import latinConfig from '../../src/scripts/latin.json';
+import mundoConfig from '../../src/scripts/mundo.json';
+
+const scriptConfig = {
```

Comment:
👋 Hey, just being nosey but It might be worth thinking about how we can tree shake/bundle split this

Reply:
@dr3 Maybe you've come across this problem before and can give us a hand. At this point, there is a 1-to-1 correspondence between the a script variable and the filename from which we will be pulling a json object. Would it be the case here to just to an fs.read for the specific file (considering it's not executable code, but just JSON)?

Reply:
Hey :) Yeah I appreciate this is a new concern, we would be happy to go over in more detail, but the jist is that doing `import mundoConfig` etc for all services causes every services config to be bundled when you may only need one.

One possible solution is to get the application to pass the config e.g. `getAtlas(scriptConfig)` and then its down to the application to load only the config it needs

Reply:
We've looked into this and implemented the possible solution below.
`require('./scripts/'+ script + '.json');`
We would add the code above to `typography.js`, in method, `getFont`.
Additionally, with this, we will need
- to disable the `import/no-dynamic-require` rule, and
- add `--copy-files` to script, `build` in the top-level `package.json`